src/pages/theming/css-shadow-parts.md

```diff
@@ -0,0 +1,153 @@
+---
+previousText: 'CSS Variables'
+previousUrl: '/docs/theming/css-variables'
+nextText: 'Colors'
+nextUrl: '/docs/theming/colors'
+---
+
+# CSS Shadow Parts
+
+CSS Shadow Parts allow developers to style CSS properties on an element inside of a shadow tree. This is extremely useful in customizing Ionic Framework <a href="https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM" target="_blank" rel="noopener noreferrer">Shadow DOM</a> components.
```

Comment:
Should we define what a "shadow tree" or "shadow root" is?

Reply:
I think it might be outside the context of this to go into detail, but maybe links to mdn or something.

Reply:
Yeah I previously had it linking to https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM but that same link is used in the next sentence so it seemed like too much. I can add it back there if we think it should be there.

Reply:
Ah ok if there's already a link then it's probs fine as is.

Reply:
I'm not sure honestly. Do people get annoyed by too many links? haha. I wasn't sure if it would annoy someone that they both link to the same place.

Reply:
The more I think about it, I agree with Mike. The term "shadow root" is not Ionic specific, and someone could pretty easily look up the definition (or click the other shadow dom link). Also, I think the expectation is that if you are using shadow parts, you should be familiar with the concept of the shadow dom in general.

Reply:
So no link?

Reply:
No link 👍